src/app/dashboard/page.tsx

```
@@ -108,369 +75,186 @@ export default function Dashboard() {
+              {/* アバター選択 */}
+              <Dialog>
+                <DialogTrigger asChild>
+                  <button className="group flex flex-col items-center gap-3 p-6 rounded-2xl hover:bg-white/60 transition-all duration-300">
+                    <div className="relative">
+                      <div className="p-4 bg-gradient-to-br from-orange-100 to-orange-200 rounded-2xl group-hover:shadow-lg transition-all duration-300 group-hover:scale-110">
+                        <UserCircle className="h-12 w-12 text-orange-600" />
                       </div>
+                    </div>
+                    <span className="text-sm font-medium text-slate-700 group-hover:text-slate-900">
+                      アバター選択
+                    </span>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)